CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# v1.4.0 (Aug 2020)
+  * Objective-C/Swift interop:
+    * Reworked exception handling ([GH-4191](https://github.com/JetBrains/kotlin-native/pull/4191))
```

Comment:
Aren't GH-* links recognized automatically?

Reply:
It may be useful for non-GitHub readers.